README.md

```diff
@@ -24,6 +24,29 @@ Make sure you have Docker installed, then run:
 docker-compose up
 ```
 
+## Local Development of `calendarium-romanum`
+
+If you need to work on the `calendarium-romanum` gem locally you can do so in the following way:
+
+1. Clone down the `calendarium-romanum` app:
+  ```sh
+  git clone git@github.com:Lumen-de-Lumine/calendarium-romanum.git calendarium-romanum
+  ```
+1. Update the following lines in the `docker-compose.yml`
+  ```sh
+  # - ./calendarium-romanum:/home/app/webapp/calendarium-romanum
+  # - .bundle_config:/home/app/webapp/.bundle/config
+  ```
+1. Modify the source of the `calendarium-romanum` and then restart via `docker-compose restart`
```

Comment:
I'd number these 1-3, so that it makes sense if you're just using a text editor.